geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -1189,11 +1189,11 @@ public void postCreateRegion() {
       for (GatewaySender sender : cache.getAllGatewaySenders()) {
         if (sender.isParallel() && allGatewaySenderIds.contains(sender.getId())) {
           /*
-           * get the ParallelGatewaySender to create the colocated partitioned region for this
+           * get the ParallelGatewaySender to create the collocated partitioned region for this
```

Comment:
This should be "colocated" with one L.

Reply:
OK, I will change that. Thanks for the feedback.